src/libs/ValidationUtils.ts

```diff
@@ -352,6 +375,68 @@ function isValidAccountRoute(accountID: number): boolean {
     return CONST.REGEX.NUMBER.test(String(accountID)) && accountID > 0;
 }
 
+/**
+ * Checks if the time input is at least one minute in the future. inputTime: '12:00 PM', inputDateTime: '2023-09-27'
+ * */
+const isTimeAtLeastOneMinuteInFuture = (inputTime: string, inputDateTime: string): boolean => {
+    if (!inputTime) {
+        return false;
+    }
+    // Parse the hour and minute from the time input
+    const [hourStr, minuteStr, period] = inputTime.split(/[:\s]+/);
+    let hour = parseInt(hourStr, 10);
+
+    if (hour === 0) {
+        return false;
+    }
+
+    // Convert 12-hour format to 24-hour format
+    if (period.toUpperCase() === 'PM' && hour !== 12) {
+        hour += 12;
+    } else if (period.toUpperCase() === 'AM' && hour === 12) {
+        hour = 0;
+    }
+
+    const minute = parseInt(minuteStr, 10);
+
+    // Check if the inputDateTime contains a time portion or is just a date
+    const dateTimeFormat = inputDateTime.includes(':') ? 'yyyy-MM-dd HH:mm:ss' : 'yyyy-MM-dd';
+
+    // Parse the input date and time (or just date) accordingly
+    const parsedInputDateTime = parse(inputDateTime, dateTimeFormat, new Date());
+
+    // Set the hour and minute from the time input to the date from the date-time input
+    const combinedDate = set(parsedInputDateTime, {hours: hour, minutes: minute});
```

Comment:
Should this be using `combineDateAndTime` from DateUtils?

Reply:
yes, we can reuse it here, thanks!